README.md

```diff
@@ -20,9 +20,25 @@
   </h3>
 </div>
 
-## Installation
+## Pre-requisites
+The current version of CLI is all-in-one package -- it is used both for production and testing. As a result, even if you
+need CLI only for the token transfer you need to install all testing dependencies. This will be changed in the future.
 
-TODO: Need to publish it to npm.
+If you want to run local Ethereum network you would need to install [ganache-cli](https://www.npmjs.com/package/ganache-cli) globally.
```

Comment:
This is not true. i put a ganache-cli as a local dependency in `vendor/ganache/package.json` and `index.js prepare` install it. Then `start ganache` start ganache with the local install